external/fv3fit/fv3fit/_shared/scaler.py

```diff
@@ -50,9 +50,9 @@ def __init__(self, std_epsilon: np.float64 = 1e-12):
                 that are constant across samples) are unable to be scaled due to
```

Comment:
Are these changes necessary? Some cleanups are nice, but it is nice to keep the diff isolated to places/files touched by the new code.

Reply:
IIRC mypy complained to me about the first two, on the third line I removed the specification because the type is over-specified.

Reply:
This added type hints are not accuracte since `None` is not a numpy array. Can you try reverting? What was the specific error?